### PR TITLE
AN-4553/deploy new defi gold views

### PR DESCRIPTION
**Pull request by forgxyz**

### CODE REVIEW

File: models/gold/defi/defi__ez_bridge_activity.sql [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+        {{ ref('silver__complete_native_prices') }}
+)
+SELECT
+    tx_id,
+    block_timestamp,
+    block_height,
+    bridge_address,
+    b.token_address,
+    p.symbol AS token_symbol,
+    gross_amount AS amount_adj,
```
Comment (eric-laurello) [inline models/gold/defi/defi__ez_bridge_activity.sql:32]: no sure what the rest of flow does, but we typically do amount and the decimal adjusted and amount_unadj as the raw value.

File: models/gold/defi/defi__fact_dex_swaps.sql [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    token_in_destination AS origin_to_address,
+    token_in_contract AS token_in,
+    token_in_amount AS amount_in_adj,
+    swaps_final_id AS fact_dex_swaps_id,
+    inserted_timestamp,
+    modified_timestamp
+FROM
+    {{ ref('silver__swaps_final') }}
+WHERE
+    token_in_contract IS NOT NULL
```
Comment (eric-laurello) [inline models/gold/defi/defi__fact_dex_swaps.sql:27]: Can we push this filter to the silver swap_final model?

File: models/gold/defi/defi__ez_bridge_activity.sql (hunk 1 above)
Comment (forgxyz) [reply models/gold/defi/defi__ez_bridge_activity.sql:32]: Ok, then just amount. The number is an int from the chain so I was thinking adjusted as a signal for it's decimal adjusted. Can just do `amount`

File: models/gold/defi/defi__fact_dex_swaps.sql (hunk 2 above)
Comment (forgxyz) [reply models/gold/defi/defi__fact_dex_swaps.sql:27]: Probably, but it was on the old gold view and I cannot remember the exact reason why. Will leave it here and note it for the silver updates post-Crescendo

**Review by eric-laurello — APPROVED** [review]

**Pull request merged**